xpk.py

```diff
@@ -2594,6 +2649,15 @@ def directory_path_type(value):
     ),
 )
 
+workload_create_parser_optional_arguments.add_argument(
+    '--deploy-stacktrace-sidecar',
+    action='store_true',
+    help=(
+        'Add this argument to deploy a sidecar container that will '
+        'read the stack traces collected in /tmp/debugging directory.'
```

Comment:
Should this also say "and forward them to Cloud Logging"?

Reply:
Sorry, I missed your comment before merging. Will update in next PR.